src/test/kotlin/io/github/mojira/arisa/modules/commands/MakePrivateCommandTest.kt

```diff
@@ -0,0 +1,37 @@
+package io.github.mojira.arisa.modules.commands
+
+import com.mojang.brigadier.exceptions.CommandSyntaxException
+import io.github.mojira.arisa.utils.mockIssue
+import io.kotest.assertions.throwables.shouldThrow
+import io.kotest.core.spec.style.StringSpec
+import io.kotest.matchers.shouldBe
+
+class MakePrivateCommandTest : StringSpec({
+    val command = MakePrivateCommand()
+
+    "should throw ALREADY_PRIVATE when the ticket is already set to private" {
+        val issue = mockIssue(
+                securityLevel = "private"
```

Comment:
(minor) the indent here (and also below) seems to be 8 spaces instead of 4 spaces

Reply:
Fixed